module/min/minwebgl/src/texture/d2.rs

```diff
@@ -2,6 +2,28 @@ use crate::*;
 
 type GL = web_sys::WebGl2RenderingContext;
 
+/// Represents a sprite sheet containing multiple sprites arranged in rows and columns.
+/// 
+/// A sprite sheet is commonly used in 2D game development to manage and optimize
+/// rendering of animations or multiple images by storing them in a single texture.
+pub struct SpriteSheet
+{
+  /// Number of sprites in each row of the sheet
+  pub sprites_in_row : u32,
+
+  /// Number of sprites in each column of the sheet
+  pub sprites_in_col : u32,

```

Comment:
You don't need this. You can find row and col of the image like this
row = i % sprites_in_row
col = i / sprites_in_row